tests/cpp/CMakeLists.txt

```diff
@@ -27,6 +27,12 @@ if(NOT DEFINED TE_LIB_PATH)
 endif()
 
 find_library(TE_LIB NAMES transformer_engine PATHS ${TE_LIB_PATH} ENV TE_LIB_PATH REQUIRED)
+
+if(EXISTS ${NVTE_MPI_INCLUDE})
+    find_library(MPI_LIB NAMES mpi PATHS ${NVTE_MPI_INCLUDE} REQUIRED)
```

Comment:
I think `NVTE_MPI_INCLUDE` is a misleading name. Something like `NVTE_MPI_LIB_DIR` would be better.